convert_results.py

```diff
@@ -0,0 +1,182 @@
+"""This script converts JSON result files to Markdown summaries.
+
+The JSON format is meant to contain all details from tests. For human readable
+outputs, use this script to extract the most interest information from your JSON
+reports and write them into an eye-friendly markdown file.
+
+  Typical usage example:
+
+  python convert_results.py --source=results/ --target=result_summaries/
+"""
+
+import argparse
+import json
+import os
+from tabulate import tabulate
+
+
+def summary_table(tests, passed_test_count, total_test_count):
```

Comment:
Your doc string do not follow google standard: you need to specify what are the args and returns. 
See: `def function_with_types_in_docstring(param1, param2):` in : 
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
Added documentation.